Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/InputActionAssetDrawer.cs

```diff
@@ -94,7 +94,7 @@ static bool IsAssetProjectWideActions(SerializedProperty property)
             if (property.objectReferenceValue != null)
             {
                 var asset = (InputActionAsset)property.objectReferenceValue;
-                isAssetProjectWideActions = asset?.name == ProjectWideActionsAsset.kAssetName;
+                isAssetProjectWideActions = asset?.name == InputSystem.kProjectWideActionsAssetName;
```

Comment:
I would recommend we delegate this comparison to ProjectWideActions class as a static function, i.e.

var isAssetProjectWideActions = InputSystem.IsProjectWideActions(property.objectReferenceValue);

Instead of having such comparisons scattered around in the code, then its more maintainable.